dist/husky.css

```diff
@@ -10562,22 +10562,29 @@ a.btn-black, a.btn-highlight {
   width: 100%;
   height: 70px;
   position: relative;
-  top: 0px;
-  left: 0px;
+  top: 0;
+  left: 0;
   color: #fff;
-  line-height: 70px;
   padding-left: 55px;
   font-size: 12px;
-  z-index: 15000;
+  z-index: 200;
```

Comment:
This was necessary to fix this Bug:
![image](https://cloud.githubusercontent.com/assets/405874/12420851/511decf8-bebf-11e5-9dfb-34aeb2fb66ae.png)

I think a z-index value that high is not required anymore, since the new labels push the content away, instead of overlapping it. Or do you know any use cases where this might make problems? /cc @wachterjohannes @chirimoya 

PS: I think we should adapt [something like this](https://css-tricks.com/handling-z-index/), adjusting z-index values is a PITA currently.